feat: add remove void from mongo step [TCTC-6102] (#1797)
ToucanToco/weaverbird: server/CHANGELOG.md

## Unreleased

> - Now we can clean mongo steps containing `__VOID__` as values with `pipeline.remove_void_conditions_from_mongo_steps()`.

Reviewer: ```suggestion
- `__VOID__` values can now be removed from mongo queries with `weaverbird.pipeline.remove_void_conditions_from_mongo_steps()`.
```

Please use simple statements for release notes
